fluid/chan.cpp

```diff
@@ -77,8 +76,13 @@ void Channel::initCtrl()
             gen[i]     = 0.0f;
             gen_abs[i] = 0;
             }
-      for (int i = 0; i < 128; i++)
+
+      /* Reset key pressure and CCs for all possible values */
+      for (int i = 0; i < 128; i++) {
+            // For MuseScore purposes, default to 80 for poly aftertouch
+            setKeyPressure(i, 80);
```

Comment:
By the way, why did you use 80 instead of 64 or some other value?

Reply:
Default dynamic for MuseScore is 80. I can't see a way of sharing some constant across fluid and libmscore, unfortunately.